Proy/game.py

```diff
@@ -1,81 +1,118 @@
+""""
++Encontre una extension para el estilo PEP8 en Visual Studio Code llamada autopep8 y tambien black formatter.
+-No encontre un atajo ni extension para python para las importaciones
++Mejoré la funcion winner usando set() y evitando mucha repeticion de (== symbol)
++Use try except para que no se rompa el programa cuando no se ingresen numeros en la funcion get_valid_input(game), ademas use .replace(",", " ").split() para que acepte datos separados como: 
+1,1
+1 ,1
+1 , 1
+1, 1
+1 1
+1     1
+*Estoy aprendiendo como funciona pytest y todavia no se como simular una entrada de usuario, logre hacer el test solo a 3 funciones
```

Comment:
Simulating user input is hard and often painful...
However, there are only two functions that require input from the user. 
Testing the other functions is difficult for another reason: They are not "pure" (this is a term from functional programming that you can read up on). While this often happens, here it is especially difficult as the state they depend on (the `game`) is not bound to a class where it could be prepared, but just floating around freely in the file. I can show you how one would change that if we have a call! :)